packages/amplify_datastore/android/proguard-rules.pro

```diff
@@ -1 +0,0 @@
--keep class io.reactivex.rxjava3.** { *; }
```

Comment:
This was needed to prevent a crash: https://github.com/aws-amplify/amplify-flutter/pull/1064

Reply:
Yea according to the comment link you added in the original fix: https://stackoverflow.com/a/64506880/12626712
This should no longer be an issue when using gradle >= v6.5, we upgraded to 7.2.

Reply:
Sweet, thanks!